### DISCUSSION TITLE
[Mac] Set transparency with shift+scrolling - 使用 shift + 滚动设置透明度

### DISCUSSION THREAD

**Discussion by JohnLockeNJ** [Snipaste/feedback#2187] [Ideas]
Could you let us adjust the transparency of a Snip by holding down the SHIFT key while scrolling, similar to how you can resize a Snip by simply scrolling?

Google Translate/谷歌翻译:
您能否让我们通过在滚动时按住 SHIFT 键来调整 Snip 的透明度，类似于您当前通过简单滚动来调整 Snip 的大小？

**Comment by Geeyun-JY3** [comment]
Currently you can customize the shortcut of adjusting the transparency: you can set it as <kbd>Shift</kbd>+<kbd>Wheel scroll</kbd>
![image](https://github.com/Snipaste/feedback/assets/49396421/c54e1d63-d22f-430f-aa7a-09a598307e02)
